File: combination/chan/test1/main.php
<?php

    $numCount = 20000;

    $start = microtime(true);

    function getAddNum($i){
        $sum = 0;
        while($i > 0){
            $sum+=$i;
            $i--;
        }
        return $sum;
    }

    $a = [];
    for($i = 1; $i <= $numCount; $i++){
        getAddNum($i);
    }

    $end = microtime(true);

    echo "used ". ($end - $start) . "s";

    ?>

File: combination/chan/test1/main.py
# from datetime import datetime

# num = 20000

# start = datetime.now().timestamp()

# def addNum(i):
#     sum = 0
#     while i > 0:
#         sum+= i
#         i-= 1
#     return sum
    
# i = 1
# while i <= num:
#     addNum(i)
#     i+= 1

# end = datetime.now().timestamp()

# print("used %.3fs" % (end - start))

import threading
import asyncio

@asyncio.coroutine
def hello():
    print('Hello world! (%s)' % threading.currentThread())
    yield from asyncio.sleep(2)
    print('Hello again! (%s)' % threading.currentThread())

loop = asyncio.get_event_loop()
tasks = [hello(), hello()]
loop.run_until_complete(asyncio.wait(tasks))
loop.close()